.github/workflows/dependabot_pr.yaml

```diff
@@ -0,0 +1,26 @@
+name: "Dependabot Auto Approve and Merge"
+
+on:
+  pull_request:
+
+permissions:
+  pull-requests: write
+  contents: write
+
+jobs:
```

Comment:
Should we put that together with the rest of our workflows?

Reply:
Ok updated this to use the workflow added in this PR:
- https://github.com/canonical/sdcore-github-workflows/pull/35